src/components/sw360/SW360Navbar/SW360Navbar.tsx

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) Siemens AG, 2023. Part of the SW360 Frontend Project.
+// Copyright (c) Helio Chissini de Castro, 2023. Part of the SW360 Frontend Project.
+
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+
+// SPDX-License-Identifier: EPL-2.0
+// License-Filename: LICENSE
+
+'use client'
+
+import Link from 'next/link'
+import { useState } from 'react'
+import { usePathname } from 'next/navigation'
+import { useTranslations } from 'next-intl'
+import Nav from 'react-bootstrap/Nav'
+
+import { COMMON_NAMESPACE } from '@/object-types/Constants'
+import { LOCALES as locales } from '@/object-types/Constants'
+
+function SW360Navbar() {
+    const [heading, setHeading] = useState('Home')
+    const pathname = usePathname()
+    const isLoginPage = locales.includes(pathname.substring(1))
```

Comment:
This line check current page is login page or not and it missing '/' path so it should be
```
locales.includes(pathname.substring(1)) || pathname.substring(1) === ''
```

Reply:
DONE